nextcord/message.py

```diff
@@ -619,6 +621,39 @@ def created_at(self) -> datetime.datetime:
         return utils.snowflake_time(self.id)
 
 
+class MessageRoleSubscription:
+    """Represents a message's role subscription information.
+
+    This is accessed through the :attr:`Message.role_subscription` attribute if the :attr:`Message.type` is :attr:`MessageType.role_subscription_purchase`.
+
+    .. versionadded:: 2.6
+
+    Attributes
+    -----------
```

Comment:
```suggestion
    Attributes
    ----------
```

Reply:
Wouldn't that make it too short?

Reply:
It wouldn't, notice how the last dash has sort of the same width as the word
![image](https://github.com/nextcord/nextcord/assets/80943664/0c719bba-0883-4408-befc-9014cb969413)

Reply:
Oh thanks GitHub.
![Screenshot_20230920_072413_Samsung Internet](https://github.com/nextcord/nextcord/assets/33902984/de0a7cda-180b-4dba-a7ab-a7da9570c8a7)